translations/en/LC_MESSAGES/messages.po

```diff
@@ -456,7 +456,7 @@ msgid "create_multiple_accounts"
 msgstr "Create multiple accounts"
 
 msgid "accounts_intro"
-msgstr "On this page you can create accounts for multiple students at the same time. There are autoamtically added to the current class, make sure the selected class is correct!. By pressing the green + on the bottom right of the page you can add extra rows. You can delete a row by pressing the corresponding red cross. Make sure no rows are empty when you press \"Create accounts\". Please keep in mind that every username needs to be unique and the password needs to be <b>at least</b> 6 characters."
+msgstr "On this page you can create accounts for multiple students at the same time. These are automatically added to the current class, so make sure the selected class is correct! By pressing the green + on the bottom right of the page you can add extra rows. You can delete a row by pressing the corresponding red cross. Make sure no rows are empty when you press \"Create accounts\". Please keep in mind that every username needs to be unique and the password needs to be <b>at least</b> 6 characters."
```

Comment:
I don't think we need such a contrived explanation for what is a fairly simple page!

"On this page you can create accounts for multiple students at the same time." => This is pretty obvious from the page itself, since you're prompted with several empty fields!

These are automatically added to the current class, make sure the selected class is correct!. => This needs to be removed, because is redundant now that they're added to only the current class.

"By pressing the green + on the bottom right of the page you can add extra rows. You can delete a row by pressing the corresponding red cross." => I think this is clear from the buttons themselves? What do you think?

"Make sure no rows are empty when you press "Create accounts". Please keep in mind that every username needs to be unique and the password needs to be at least 6 characters. => In my opinion this is the only part worth keeping, but still maybe is better to enforce this from the backend itself instead of asking this to our users.

Reply:
I agree, I think this text can be simplified.  I'll defer to @Felienne on the verbiage.